packages/idx/src/property/search.component.html

```diff
@@ -165,7 +166,7 @@ <h2 data-ng-show="widgetName" data-ng-bind="widgetName"></h2>
             <a class="open-filters-link"
                data-ng-href="{{ advancedSearchUrl }}">
                 <md-icon md-svg-src="{{ localDir + 'images/icons/actionButtons/filters.svg' }}"></md-icon>
-                <span class="btn-text"><span data-ng-bind="advancedSearchLinkName"></span>
+                <span class="btn-text"><span data-ng-bind="advancedSearchLinkName"></span></span>
```

Comment:
Note: You have a double closing tag.  I'm not sure if that's intentional.

Reply:
theres two spans here, so it needs to be closed twice (IDE was whining to me)

Reply:
if anything, these should probably be consolidated into a single span